engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -320,14 +319,14 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     ()
   }
 
-  def makeFetchCachedResultsActor(metaInfoId: MetaInfoId, taskOutputs: Seq[TaskOutput]): Unit = {
-    context.actorOf(FetchCachedResultsActor.props(metaInfoId, self, new CallCache(SingletonServicesStore.databaseInterface)))
+  def makeFetchCachedResultsActor(callCachingEntryId: CallCachingEntryId, taskOutputs: Seq[TaskOutput]): Unit = {
+    context.actorOf(FetchCachedResultsActor.props(callCachingEntryId, self, new CallCache(SingletonServicesStore.databaseInterface)))
     ()
   }
 
-  private def fetchCachedResults(taskOutputs: Seq[TaskOutput], metaInfoId: MetaInfoId) = {
-    makeFetchCachedResultsActor(metaInfoId, taskOutputs)
-    goto(FetchingCachedOutputsFromDatabase)
+  private def fetchCachedResults(taskOutputs: Seq[TaskOutput], callCachingEntryId: CallCachingEntryId, data: ResponsePendingData) = {
+    makeFetchCachedResultsActor(callCachingEntryId, taskOutputs)
+    goto(FetchingCachedOutputsFromDatabase) using data
```

Comment:
Is this change intended? Just moving the `using` inside this function (from line 134) this would have given `using data.withCacheHit(hit).popCacheHitId._2`

EDIT: Just saw this is called with the NEW data from line 366?